src/phases/mystery-encounter-phases.ts

```diff
@@ -108,9 +109,9 @@ export class MysteryEncounterPhase extends Phase {
         }
 
         if (title) {
-          this.scene.ui.showDialogue(text, title, null, nextAction, 0, i === 0 ? 750 : 0);
+          this.scene.ui.showDialogue(text, title, null, nextAction, 0, i === 0 ? 300 : 0);
         } else {
-          this.scene.ui.showText(text, null, nextAction, i === 0 ? 750 : 0, true);
+          this.scene.ui.showText(text, null, nextAction, i === 0 ? 300 : 0, true);
```

Comment:
350 seems to be a `reused` number. Magic number?

Reply:
fixed